packages/api/tools/osiris/deploy-osiris-files.js

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const child_process = require("child_process");
 
+// scalingo app url
 const appName = process.argv[2];
+// i.e : "requests" | "actions" | "evaluations"
 const importType = process.argv[3];
```

Comment:
On peut le mettre comme type plutôt que dans un commentaire dans ce cas-ci
```suggestion
const importType: "requests" | "actions" | "evaluations" = process.argv[3];
```
ou
```suggestion
const importType = process.argv[3] as "requests" | "actions" | "evaluations";
```

Reply:
c'est du `.js`!

Reply:
Bien vu ^^